Plugins/Tweaks/Tweaks/AdjustCasterLevel.cpp

```diff
@@ -0,0 +1,175 @@
+#include "Tweaks/AdjustCasterLevel.hpp"
+
+#include "Services/Hooks/Hooks.hpp"
+#include "Utils.hpp"
+
+#include "API/CNWRules.hpp"
+#include "API/CNWSCreatureStats.hpp"
+#include "API/Globals.hpp"
+#include "API/CTwoDimArrays.hpp"
+#include "API/C2DA.hpp"
+
+
+namespace Tweaks {
+
+using namespace NWNXLib;
+using namespace NWNXLib::API;
+
+std::vector<uint8_t> AdjustCasterLevel::s_classCasterType;
+std::vector<std::tuple<uint8_t, uint8_t>> AdjustCasterLevel::s_divModClasses;
```

Comment:
This can just be a `std::vector<uint8_t>` for the mods, and avoid the extra logic of tuples.
Or even just `uint8_t[Constants::MAX_CLASSES]`. Same for other vectors.